observability/logging/logging-6.2/log6x-configuring-lokistack-otlp-6.2.adoc

```diff
@@ -0,0 +1,146 @@
+
+[NOTE]
+====
+Both global and per-tenant OTLP configurations can map attributes to stream labels.
+====
+
+Stream labels derive only from resource-level attributes, which the `LokiStack` resource structure reflects:
+
+[source,yaml]
+----
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          resourceAttributes:
+          - name: "k8s.namespace.name"
+          - name: "k8s.pod.name"
+          - name: "k8s.container.name"
+----
+
+You can drop attributes of type resource, scope or log from the log entry.
+
+[source,yaml]
+----
+# ...
+spec:
+  limits:
+    global:
+      otlp:
+        streamLabels:
+          # ...
+        drop:
+          resourceAttributes:
+          - name: "process.command_line"
+          - name: "k8s\\.pod\\.labels\\..+"
+            regex: true
+          scopeAttributes:
+          - name: "service.name"
+          logAttributes:
+          - name: "http.route"
+----
+
+[TIP]
+====
+Use regular expressions by setting `regex: true` to apply a configuration for attributes with similar names.
+====
+
+[IMPORTANT]
+====
+Avoid using regular expressions for stream labels, as this can increase data volume.
+====
+
+Attributes that are not explicitly set as stream labels or dropped from the entry are saved as structured metadata by default.
+
+[id="customizing-openshift-defaults_{context}"]
+=== Customizing OpenShift defaults
+
+In `openshift-logging` mode, certain attributes are required and cannot be removed from the configuration due to their role in OpenShift functions. Other attributes, labeled *recommended*, might be dropped if performance is impacted. For information about the attributes, see link:https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.1/log6x-opentelemetry-data-model-6.1.html#attributes_log6x-opentelemetry-data-model-6-1[OpenTelemetry data model attributes].
+
+When using the `openshift-logging` mode without custom attributes, you can achieve immediate compatibility with OpenShift tools. If additional attributes are needed as stream labels or some attributes need to be droped, use custom configuration. Custom configurations can merge with default configurations.
+
+[id="removing-recommended-attributes_{context}"]
+=== Removing recommended attributes
+
+To reduce default attributes in `openshift-logging` mode, disable recommended attributes:
+
+[source,yaml]
+----
+# ...
+spec:
+  tenants:
+    mode: openshift-logging
+    openshift:
+      otlp:
+        disableRecommendedAttributes: true # <1>
+----
+<1> Set `disableRecommendedAttributes: true` to remove recommended attributes, which limits default attributes to the required attributes or stream labels.
+
+[NOTE]
+====
+This setting might negatively impact query performance, as it removes default stream labels. You should pair this option with a custom attribute configuration to retain attributes essential for queries.
+====
```

Comment:
```suggestion
<1> Set `disableRecommendedAttributes: true` to remove recommended attributes, which limits default attributes to the required attributes or stream labels.
+
[NOTE]
====
This setting might negatively impact query performance, as it removes default stream labels. You must pair this option with a custom attribute configuration to retain attributes essential for queries.
====
```

- Try adding `+` so that this note is explicitly a part of <1> because it's related to the `...:true` setting.
- Avoid using should: [guideline](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s)

Reply:
done